Project.toml

```diff
@@ -24,6 +24,7 @@ Lux = "b2108857-7c20-44ae-9111-449ecde12c47"
 LuxCUDA = "d0bbae9a-e099-4d5b-a835-1c6931763bda"
 LuxCore = "bb33d45b-7691-41d6-9220-0943567d0623"
 NNlib = "872c559c-99b0-510c-b3b7-b6c96a88d5cd"
+#NeuralClosure = "099dac27-d7f2-4047-93d5-0baee36b9c25"
```

Comment:
Any suggestion on how we should handle this? every time that i open a PR I end up add it back to Project.toml :sweat_smile:

Reply:
I think there are two options:
1. Add the neuralclosure lib to our package, or the package repository
2. manually add neuralclosure in our github pipelines

Reply:
I think it is better to have this file up to date, especially because you need to update it sometimes, especially with the new compat section. I will create an issue